JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -195,11 +196,11 @@ private fun TopicItem(topicKey: String, itemTitle: String) {
             // TODO(b/150060763): Remove box after "Bug in ripple + modifiers."
             Box {
                 Row(
-                    modifier = LayoutPadding(left = 16.dp, top = 0.dp, right = 16.dp, bottom = 0.dp)
+                    modifier = LayoutPadding(start = 16.dp, top = 0.dp, end = 16.dp, bottom = 0.dp)
                 ) {
                     Container(modifier = LayoutGravity.Center + LayoutSize(56.dp, 56.dp)) {
                         Clip(RoundedCornerShape(4.dp)) {
-                            DrawImage(image)
+                            SimpleImage(image = image)
```

Comment:
Ditto.

Reply:
This one is required since Clip needs to be bounded by the modifier to Container. Once Image component lands this can be reduced to one layout node.